src/SdBus.cpp

```diff
@@ -260,4 +260,49 @@ sd_bus* SdBus::sd_bus_flush_close_unref(sd_bus *bus)
     return ::sd_bus_flush_close_unref(bus);
 }
 
+int SdBus::sd_bus_query_sender_creds(sd_bus_message *m, uint64_t mask, sd_bus_creds **c)
+{
+    return ::sd_bus_query_sender_creds(m, mask, c);
+}
```

Comment:
Would you please decorate all these new methods with `std::lock_guard lock(sdbusMutex_);`, just like (most of the) sd-bus functions above in the file? These sd-bus calls must be thread-safe.

Reply:
Thanks for the tip, fixed.